README.md

```diff
@@ -1,72 +1,66 @@
-# GreenScout-Backend
-The backend/server software for FRC team 1816's scouting app, GreenScout
+# GreenScout Backend
+The backend/server software for FRC team 1816's scouting app, GreenScout.
 
-### IMPORTANT: IT'S FINE THAT SERVER.CRT AND SERVER.KEY ARE PUBLIC. THEY ARE ONLY USED FOR MOCKING HTTPS ON LOCALHOST FOR TESTING
-# Getting Started
+**IMPORTANT: IT'S FINE THAT SERVER.CRT AND SERVER.KEY ARE PUBLIC. THEY ARE ONLY USED FOR MOCKING HTTPS ON LOCALHOST FOR TESTING** (this will be fixed...)
 
-It's expected that you at least know basic programming principles and practices. Additionally, knowing the [Go Programming Language](https://go.dev/learn/) will be neccessary when contributing to this CLI.
+## Getting Started
 
-To get started, you'll first need the [Go Programming Language](https://go.dev/dl/), [Git](https://git-scm.com/downloads), and [VS Code (Optional)](https://code.visualstudio.com/Download).
+It's expected that you at least know basic programming principles and practices. Additionally, knowing the [Go Programming Language](https://go.dev/learn/) will be neccessary when contributing to this project. A (relatively) short primer on Go is provided [here](docs/Go.md).
 
-Additional dependencies include [Sqlite 3](https://sqlite.org/download.html), [The Python programming language](https://www.python.org/downloads/), and [pip](https://pypi.org/project/pip/)
+[VS Code](https://code.visualstudio.com/Download) is the recommended editor of choice for this project, along with installing it's [Golang Extension](https://marketplace.visualstudio.com/items?itemName=golang.Go).
 
-Once you have all that installed, open up your terminal and enter this command
```

Comment:
The reformatting here deleted the explanation for what you're trying to accomplish with the clone. I know it should be obvious what's going on there, but it would be good to put in some kind of verbal indicator that you're starting specific instructions for using a terminal to clone the repo and set up the dev environment.

Reply:
So insert something at (the new) line 20 like:

```markdown
Once you have all that installed, you'll need to actually download the code. While the rest of this document will detail how to do it from a command line terminal, feel free to clone and open this git repo by your own method of choice.
```

Reply:
Yes, much like this.